ui/dashboards/src/components/DashboardToolbar.tsx

```diff
@@ -28,6 +28,7 @@ export const DashboardToolbar = (props: DashboardToolbarProps) => {
 
   const { isEditMode, setEditMode } = useEditMode();
   const { addPanelGroup, addPanel } = useDashboardActions();
+  const isLaptopSize = useMediaQuery(useTheme().breakpoints.up('sm'));
```

Comment:
I am wondering if it makes the create a wrapper around that. So it's even more easier to use it, even if it's already super easy to use !